lighthouse/tests/beacon_node.rs

```diff
@@ -0,0 +1,844 @@
+use beacon_node::ClientConfig as Config;
```

Comment:
To make CI run faster we can combine all the integration tests into a single binary (see https://endler.dev/2020/rust-compile-times/#combine-all-integration-tests-in-a-single-binary)

If you add `lighthouse/tests/main.rs` with a `mod` for each file, I think that should work (I tested just now with a smaller crate). i.e.

```rust
mod account_manager;
mod beacon_node;
mod validator_client;
```